src/libs/sfdk/buildengine.cpp

```diff
@@ -391,11 +391,7 @@ void BuildEnginePrivate::enableUpdates()
 
     virtualMachine->setAutoConnectEnabled(true);
 
-    VirtualBoxManager::fetchVirtualMachineInfo(virtualMachine->name(),
-            VirtualBoxManager::NoExtraInfo, q, [=](const VirtualMachineInfo &info, bool ok) {
-        QTC_ASSERT(ok, return);
-        updateVmProperties(info);
-    });
+    updateVmProperties(q, [](bool ok) { Q_UNUSED(ok); });
```

Comment:
Q_UNUSED does not need a semicolon -> removing it would get rid of a compiler warning

Reply:
Fixed. Interesting, which compiler it is? Only with that I noticed I didn't use -Wall -Werror for sfdk lib (fixed), but even after enabling that, g++-8 doesn't issue a warning here.

Reply:
clang